lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -614,8 +615,8 @@ def push_student_submissions_to_s3(_xmodule_instance_args, _entry_id, course_id,
         TASK_LOG.error(e.message)
         return "failed"
 
-    header, datarows = student_submissions(course)
-    rows = [header] + datarows
+    header = ["Section", "Subsection", "Unit", "Problem", "Order In Course", "Location", "Student", "Response"]
```

Comment:
We're also defining this list in ` lms/djangoapps/instructor/management/commands/dump_student_submissions.py`.

Could/should this be constant, defined once in a single location to reduce risk of drift/typos?